### ISSUE TITLE
Bug: uncaught exception Error: ResizeObserver loop limit exceeded

### ISSUE THREAD

**Issue by hsablonniere** [CleverCloud/clever-components#236]
```
uncaught exception Error: ResizeObserver loop limit exceeded
```

**Comment by hsablonniere** [comment]
Does it mean, we need to throttle?

**Comment by hsablonniere** [comment]
We could try the lit lab resizer observer reactive controller and see if it improves the situation.

**Comment by pdesoyres-cc** [comment]
@hsablonniere,  Do you remember in which context this error occurs?

Note 1. We tried to use lit lab resizer but it doesn't fix the problem (see #551).

Note 2. We already know that we have this error during unit tests and it can be ignored since the unit test are run so fast that the error can be considered false positive (see #761).

**Comment by hsablonniere** [comment]
I took some time to dig the subject and created [another issue](https://github.com/CleverCloud/clever-components/issues/808) with the details, the solutions and a proposition. Let's continue the discussion there.
